sdk/translation/ai-translation-text/test/public/dictionaryExamplesTest.spec.ts

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Recorder } from "@azure-tools/test-recorder";
+import { assert } from "chai";
+import {
+  DictionaryExampleItemOutput,
+  DictionaryExampleTextItem,
+  LookupDictionaryExamplesQueryParamProperties,
+  TextTranslationClient,
+} from "../../src";
+import { createTranslationClient, startRecorder } from "./utils/recordedClient";
+import { Context } from "mocha";
+
+describe("DictionaryExamples tests", () => {
+  let recorder: Recorder;
+  let client: TextTranslationClient;
+
+  beforeEach(async function (this: Context) {
+    recorder = await startRecorder(this);
+    client = await createTranslationClient({ recorder });
+  });
+
+  afterEach(async function () {
+    await recorder.stop();
+  });
+
+  it("single input element", async () => {
+    const inputText: DictionaryExampleTextItem[] = [{ text: "fly", translation: "volar" }];
+    const parameters: LookupDictionaryExamplesQueryParamProperties & Record<string, unknown> = {
+      to: "es",
+      from: "en",
+    };
+    const response = await client.path("/dictionary/examples").post({
+      body: inputText,
+      queryParameters: parameters,
+    });
+    if (response.status !== "200") {
+      throw response.body.toString();
+    }
+    assert.equal(response.status, "200");
+    const dictionaryExamples = response.body as DictionaryExampleItemOutput[];
+    assert.isTrue(dictionaryExamples[0].normalizedSource === "fly");
```

Comment:
nitpick: use `equals` instead?
```suggestion
    assert.equals(dictionaryExamples[0].normalizedSource, "fly");
```